src/main/catalog-sources/general.ts

```diff
@@ -67,6 +67,6 @@ const generalEntities = observable([
   preferencesEntity
 ]);
 
-export function initializeGeneralEntities() {
+export function syncGeneralEntities() {
```

Comment:
We are not initializing anything here so I changed this function name (and the one for weblinks) and removed the exports from `./initializers/index.ts`